applications/Unity.GrantManager/src/Unity.GrantManager.Domain.Shared/Localization/GrantManager/en.json

```diff
@@ -303,6 +300,11 @@
     "ApplicantInfoView:ApplicantInfo.Province": "Province",
     "ApplicantInfoView:ApplicantInfo.PostalCode": "Postal Code",
 
+    "FundingAgreementInfoView:FundingAgreementInfoTitle": "Funding Agreement Info",

```

Comment:
Please Rename the tab as requested in the Ticket and commented by Maryam
I believe now she would like the Label inside the tab to have the Info but the tab name to not have Info

Reply:
Great job on the first ticket

Reply:
@jimmyPasta  I have updated the label to "Funding Agreement" in this feature branch.